docs/cookbook/html-conversion.md

```diff
@@ -0,0 +1,980 @@
+        type: 'string',
+        label: 'Heading'
+      },
+      subheading: {
+        type: 'string',
+        label: 'Subheading'
+      },
+      headerImage: {
+        type: 'area',
+        required: true,
+        options: {
+          widgets: {
+            '@apostrophecms/image': {}
+          },
+          max: 1
+        }
+      },
+      main: {
+        type: 'area',
+        options: {
+          widgets: {
+            '@apostrophecms/rich-text': {
+              toolbar: [
+                'styles',
+                '|',
+                'bold',
+                'italic',
+                'strike',
+                'link',
+                '|',
+                'bulletList',
+                'orderedList'
+              ],
+              styles: [
+                {
+                  tag: 'p',
+                  label: 'Paragraph (P)'
+                },
+                {
+                  tag: 'h3',
+                  label: 'Heading 3 (H3)'
+                },
+                {
+                  tag: 'h4',
+                  label: 'Heading 4 (H4)'
+                }
+              ]
+            },
+            '@apostrophecms/image': {},
+            '@apostrophecms/video': {}
+          }
+        }
+      }
+    },
+    group: {
+      basics: {
+        label: 'Basics',
+        fields: [
+          'title',
+          'heading',
+          'subheading',
+          'headerImage',
+          'main'
+        ]
+      }
+    }
+  }
+};
+
+```
+
+<template v-slot:caption>
+modules/default-page/index.js
+</template>
+</AposCodeBlock>
+
+To accommodate the 'Contact Us' page, we could also the widgets from the [form extension](https://apostrophecms.com/extensions/form-builder-3-x) in the `main` area. Since we are using the `default-page` which already exists in our project we don't need to modify either the `app.js` or `modules/@apostrophecms/page/index.js` files.
+
+### Modifying the logged-in page display
+If we were to take a look at our page right now while logged-in as an editor, we would see a couple of problems. First, the navigation section is styled to be added at the top of the page using a `postion: absolute` CSS rule. The problem with this is that this ends up putting our navigation *over* the ApostropheCMS admin-bar. Not only can we not see the navigation, but this also blocks access to the admin-bar menus. So, we need to add some code onto the page that will move our navigation below the admin-bar in the page flow.
+
+There are several areas in our project where we could add code to solve this problem. In this case, we will add a small script to our asset module again. While we could add it to `modules/@apostrophecms/asset/ui/src/index.js` along with the template code, this would result in the delivery of extra unnecessary JavaScript to all users. Instead, we will add the code into `modules/@apostrophecms/asset/ui/apos/apps`. This folder is commonly used in projects to add new custom Vue UI components.
```

Comment:
```suggestion
There are several areas in our project where we could add code to solve this problem. In this case, we will add a small script to our asset module again. While we could add it to `modules/@apostrophecms/asset/ui/src/index.js` along with the template code, this would result in the delivery of extra unnecessary JavaScript to all users. Instead, we will add the code into `modules/@apostrophecms/asset/ui/apos/apps`. This folder is commonly used in projects to add new custom Vue UI components and is only served to logged in users.
```

Maybe add a little more detail here so it's clear why `apos/apps` vs `ui/src`.

Reply:
Changed.